flake/dev/generate-all-maintainers/generate-all-maintainers.py

```diff
@@ -0,0 +1,208 @@
+            "This may not work correctly with flakes.",
+            file=sys.stderr,
+        )
+        # Assumes this script is at: <root>/flake/dev/generate-all-maintainers/
+        return Path(__file__).parent.parent.parent.parent.resolve()
+
+
+class MetaMaintainerGenerator:
+    """Generates maintainers list using meta.maintainers from nixvim evaluation."""
+
+    def __init__(self, nixvim_root: Path):
+        self.nixvim_root = nixvim_root
+        self.nixvim_maintainers_file = nixvim_root / "lib" / "maintainers.nix"
+        self.output_file = nixvim_root / "generated" / "all-maintainers.nix"
+        self.extractor_script = (
+            nixvim_root
+            / "flake"
+            / "dev"
+            / "generate-all-maintainers"
+            / "extract-maintainers-meta.nix"
+        )
+
+    def extract_maintainers_from_meta(self) -> dict:
+        """Extract maintainer information using meta.maintainers."""
+        print("🔍 Extracting maintainers using meta.maintainers...")
+
+        try:
+            result = subprocess.run(
+                [
+                    "nix",
+                    "eval",
+                    "--file",
+                    str(self.extractor_script),
+                    "--json",
+                ],
+                capture_output=True,
+                text=True,
+                timeout=60,
+            )
+
+            if result.returncode == 0:
+                data = json.loads(result.stdout)
+                print("✅ Successfully extracted maintainers using meta.maintainers")
+                return data
+            else:
+                print(f"❌ Failed to extract maintainers: {result.stderr}")
+                sys.exit(1)
+
+        except subprocess.TimeoutExpired:
+            print("❌ Timeout while extracting maintainers")
+            sys.exit(1)
+        except Exception as e:
+            print(f"❌ Error extracting maintainers: {e}")
+            sys.exit(1)
+
+    def generate_maintainers_file(self) -> None:
+        """Generate the complete all-maintainers.nix file."""
+        print("📄 Generating all-maintainers.nix using meta.maintainers...")
+
+        # Extract maintainers using meta.maintainers
+        maintainer_data = self.extract_maintainers_from_meta()
+
+        nixvim_maintainers = maintainer_data["categorized"]["nixvim"]
+        nixpkgs_maintainers = maintainer_data["categorized"]["nixpkgs"]
+        formatted_maintainers = maintainer_data["formatted"]
+
+        print(f"🏠 Nixvim maintainers: {len(nixvim_maintainers)}")
+        print(f"📦 Nixpkgs maintainers: {len(nixpkgs_maintainers)}")
+
+        with open(self.output_file, "w") as f:
+            f.write("""# Nixvim all maintainers list.
+#
+# This file lists all referenced maintainers in Nixvim.
+#
+# This file is automatically generated using meta.maintainers from nixvim evaluation
+# DO NOT EDIT MANUALLY
+#
+# To regenerate: nix run .#generate-all-maintainers
+#
+""")
```

Comment:
We can use [`inspect.cleandoc()`](https://docs.python.org/3/library/inspect.html#inspect.cleandoc) or [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) to allow us to format this nicer:


```suggestion
            f.write(inspect.cleandoc("""
              # Nixvim all maintainers list.
              #
              # This file lists all referenced maintainers in Nixvim.
              #
              # This file is automatically generated using meta.maintainers from nixvim evaluation
              # DO NOT EDIT MANUALLY
              #
              # To regenerate: nix run .#generate-all-maintainers
              #
            """))
```

See https://stackoverflow.com/a/48112903 for the subtle differences between `cleandoc` and `dedent`.